nuxt/index.js

```diff
@@ -0,0 +1,32 @@
+/*
+  Nuxt.js module for bootstrap-vue
```

Comment:
`Nuxt.js module for `~`bootstrap-vue`~`semantic-ui-vue`

Reply:
Oops, yes, i just took a code from bootstrap-vue and didnt noticed that)